{"name": "Validate Successful Form Submission -- @1.2 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: invalid session id\nStacktrace:\n\tGetHandleVerifier [0x00007FF644B482B2+55298]\n\t(No symbol) [0x00007FF644AB5E02]\n\t(No symbol) [0x00007FF64497045D]\n\t(No symbol) [0x00007FF6449A62E9]\n\t(No symbol) [0x00007FF6449A7964]\n\tGetHandleVerifier [0x00007FF644EC0AAB+3694587]\n\tGetHandleVerifier [0x00007FF644F1728E+4048862]\n\tGetHandleVerifier [0x00007FF644F0F173+4015811]\n\tGetHandleVerifier [0x00007FF644BE47D6+695590]\n\t(No symbol) [0x00007FF644AC0CE8]\n\t(No symbol) [0x00007FF644ABCF34]\n\t(No symbol) [0x00007FF644ABD062]\n\t(No symbol) [0x00007FF644AAD3A3]\n\tBaseThreadInitThunk [0x00007FFD2644257D+29]\n\tRtlUserThreadStart [0x00007FFD2686AA58+40]\n\n", "trace": "  File \"C:\\Users\\ivich\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\ivich\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\common_steps.py\", line 24, in closeBrowser\n    context.driver.close()\n  File \"C:\\Users\\ivich\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 458, in close\n    self.execute(Command.CLOSE)\n  File \"C:\\Users\\ivich\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\ivich\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Chrome browser is Launched", "status": "passed", "start": 1701307550799, "stop": 1701307552223}, {"name": "When Open Toyster Register Account page", "status": "passed", "start": 1701307552224, "stop": 1701307554973}, {"name": "Then Input multiple \"May\" and \"BLANK\" and \"may_goh0@outlook.com\" and \"p@ssword\"", "status": "passed", "start": 1701307554974, "stop": 1701307560571}, {"name": "And Click Create", "status": "passed", "start": 1701307560571, "stop": 1701307565648}, {"name": "Then Navigate to My Account Page", "status": "passed", "start": 1701307565648, "stop": 1701307572940}, {"name": "Then Verify Account is Created", "status": "passed", "start": 1701307572940, "stop": 1701307573067}, {"name": "And Close Chrome browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: invalid session id\nStacktrace:\n\tGetHandleVerifier [0x00007FF644B482B2+55298]\n\t(No symbol) [0x00007FF644AB5E02]\n\t(No symbol) [0x00007FF64497045D]\n\t(No symbol) [0x00007FF6449A62E9]\n\t(No symbol) [0x00007FF6449A7964]\n\tGetHandleVerifier [0x00007FF644EC0AAB+3694587]\n\tGetHandleVerifier [0x00007FF644F1728E+4048862]\n\tGetHandleVerifier [0x00007FF644F0F173+4015811]\n\tGetHandleVerifier [0x00007FF644BE47D6+695590]\n\t(No symbol) [0x00007FF644AC0CE8]\n\t(No symbol) [0x00007FF644ABCF34]\n\t(No symbol) [0x00007FF644ABD062]\n\t(No symbol) [0x00007FF644AAD3A3]\n\tBaseThreadInitThunk [0x00007FFD2644257D+29]\n\tRtlUserThreadStart [0x00007FFD2686AA58+40]\n\n", "trace": "  File \"C:\\Users\\ivich\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\ivich\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\common_steps.py\", line 24, in closeBrowser\n    context.driver.close()\n  File \"C:\\Users\\ivich\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 458, in close\n    self.execute(Command.CLOSE)\n  File \"C:\\Users\\ivich\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\ivich\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1701307573067, "stop": 1701307573129}], "parameters": [{"name": "firstName", "value": "May"}, {"name": "lastName", "value": "BLANK"}, {"name": "email", "value": "may_goh0@outlook.com"}, {"name": "password", "value": "p@ssword"}], "start": 1701307550799, "stop": 1701307573129, "uuid": "87789b00-47e6-418a-97e5-105cc0a9b32f", "historyId": "d316b8e793f931ae5dd9a55ff4ac9916", "fullName": "Verify User Account Creation: Validate Successful Form Submission", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Verify User Account Creation"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}